app/controllers/comfy/admin/cms/seos_controller.rb

```diff
@@ -1,4 +1,10 @@
 class Comfy::Admin::Cms::SeosController < Comfy::Admin::Cms::BaseController
   def index
   end
+
+  def check
+    webpage = 'http://www.ita-online.info'
```

Comment:
yes that is the idea, when all is integrated in the sofa or love seat. For now i try to keep my files as independent as possible, so its easier to integrate later on.